documentation/urls.py

```diff
@@ -4,9 +4,11 @@
 from documentation import views as page_views
 
 urlpatterns = [
-    path('home-documentation', page_views.HomeDocView.as_view(), name='home-documentation'),
+    path('schedule-documentation', page_views.scheduleDocView.as_view(), name='schedule-documentation'),
     path('database-documentation', page_views.DBDocView.as_view(), name='database-documentation'),
-    path('admin-documentation', page_views.AdminDocView.as_view(), name='admin-documentation'),
+    path('datasheet-documentation', page_views.datasheetDocView.as_view(), name='datasheet-documentation'),
```

Comment:
incorrect naming scheme. All classes must be written AsClassNamesAre

Reply:
will change